service-proxy-examples/service-provider/pom.xml

```diff
@@ -7,76 +7,72 @@
   <parent>
     <groupId>io.vertx</groupId>
     <artifactId>service-proxy-examples</artifactId>
-    <version>3.5.2</version>
+    <version>3.5.1.redhat-004</version>
   </parent>
 
   <artifactId>service-provider</artifactId>
 
+  <properties>
+    <vertx.skip>false</vertx.skip>
+  </properties>
+
   <dependencies>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-core</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-web</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-service-proxy</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-rx-java</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-lang-groovy</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-lang-js</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-lang-ruby</artifactId>
-      <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-codegen</artifactId>
-      <version>${project.version}</version>
     </dependency>
```

Comment:
I don't know if this dep is being used as well (it might be, if there are some .js files that need to be generated for service proxy to function properly).

Reply:
This dep is used for generating ProcessorServiceVertxEBProxy (and others) and this object is used java service provider example